ote/src/cljs/ote/views/service_viewer.cljs

```diff
@@ -0,0 +1,464 @@
+
+(defn format-descriptions
+  [data]
+  (reduce
+    (fn [new-collection item]
+      (let [lang (keyword (string/lower-case (::t-service/lang item)))
+            val (::t-service/text item)]
+        (assoc new-collection lang val)))
+    {}
+    data))
+
+(def columns (last (s/describe ::t-operator/transport-operator)))
+
+(defn spacer
+  []
+  [:hr {:style {:margin "1.5rem 0"
+                :border-bottom 0}}])
+(defn info-sections-1-col
+  ([title sv]
+   [info-sections-1-col title sv {}])
+  ([title sv settings]
+   [:div
+    (if (:sub-title settings)
+      [:p {:style {:text-transform "uppercase"
+                   :font-weight "bold"}} title]
+      [:h4 title])
+    [:div.info-block (stylefy/use-style service-viewer/info-container)
+     [:div (stylefy/use-sub-style service-viewer/info-container :left-block)
+      sv]]]))
+
+
+(defn info-sections-2-cols
+  ([title lv rv]
+   [info-sections-2-cols title lv rv {}])
+  ([title lv rv settings]
+   [:div
+    (if (:sub-title settings)
+      [:p {:style {:text-transform "uppercase"
+                   :font-weight "bold"}} title]
+      [:h4 title])
+    [:div.info-block (stylefy/use-style service-viewer/info-container)
+     [:div (stylefy/use-sub-style service-viewer/info-container :left-block)
+      lv]
+     [:div (stylefy/use-sub-style service-viewer/info-container :right-block)
+      rv]]]))
+
+(defn info-sections-3-cols
+  ([title lv mv rv]
+   [info-sections-3-cols title lv mv rv {}])
+  ([title lv mv rv settings]
+   [:div
+    (if (:sub-title settings)
+      [:p {:style {:text-transform "uppercase"
+                   :font-weight "bold"}} title]
+      [:h4 title])
+    [:div.info-block (stylefy/use-style service-viewer/info-container)
+     [:div (stylefy/use-sub-style service-viewer/info-container :left-block)
+      lv]
+     [:div (stylefy/use-sub-style service-viewer/info-container :middle-block)
+      mv]
+     [:div (stylefy/use-sub-style service-viewer/info-container :right-block)
+      rv]]]))
+
+
+
+(def open-in-new-icon
+  (ic/action-open-in-new {:style {:width 20
+                                  :height 20
+                                  :margin-right "0.5rem"
+                                  :color colors/primary}}))
+
+(defn- service-header
+  [service-name o-id s-id]
+  [:section
+   [common-ui/linkify "/#/services" [:span [icons/arrow-back {:position "relative"
+                                                              :top "6px"
+                                                              :padding-right "5px"
+                                                              :color style-base/link-color}]
+                                     (tr [:service-search :back-link])]]
+   [:h1 service-name]
+   [link-with-icon open-in-new-icon (str "/export/geojson/" o-id "/" s-id) (tr [:service-viewer :open-in-geojson])]])
+
+(defn- operator-info
+  [title operator]
+  (let [address (::t-operator/visiting-address operator)]
```

Comment:
We should only show company name, www address and business id in this view. It is confusing to show different data for different users. It will cause problems.

Reply:
This wasn't shown anywhere, but I removed the let.